actors/evm/src/interpreter/precompiles/fvm.rs

```diff
@@ -116,7 +116,7 @@ pub(super) fn get_randomness<RT: Runtime>(
 }
 
 /// Read BE encoded low u64 ID address from a u256 word
-/// Looks up and returns the encoded f4 addresses of an ID address, returning empty array if not found
+/// Looks up and returns the encoded f4 addresses of an ID address. Empty array if not found or `InvalidInput` input was larger 2^64.
```

Comment:
Is this the final form of the docs? They look a little bit unstructured. Would suggest to:

1. Explain what these methods do on a conceptual level.
2. Then specify the call layout in a way similar to what I did in [FIP-0054](https://github.com/filecoin-project/FIPs/pull/569).
3. Then specify the return value.
4. Then specify the errors.

Can fix in a follow-up PR.

Reply:
tracked in https://github.com/filecoin-project/ref-fvm/issues/1432